src/CommandingOfficers/Modifiers/UnitInstanceFilter.java

```diff
@@ -0,0 +1,27 @@
+package CommandingOfficers.Modifiers;
+
+import java.util.ArrayList;
+
+import Engine.UnitMods.UnitModFilter;
+import Engine.UnitMods.UnitModifier;
+import Units.Unit;
+import Units.UnitContext;
+
+public class UnitInstanceFilter extends UnitModFilter
+{
+  private static final long serialVersionUID = 1L;
+
+  public final ArrayList<Unit> instances = new ArrayList<>();
```

Comment:
It would probably make for a cleaner design to have a method to add more units instead of just directly accessing the public collection.

Reply:
Would it?
There's no reason to expect UIF to be modifying the input in any way; the instantiating class should have pretty complete control of the content.
I guess I should probably use a `HashSet` instead of an `ArrayList`, but those have a common external interface for adding stuff anyway.
Once it goes out of the scope of the creating function and is handled only by its interface, then it doesn't matter if the collection's public or not.
And, finally, it adds more boilerplate code and error checks to write.

Reply:
It would avoid requiring knowledge of the object internals (and uncertainty about whether direct modification is kosher). Probs not a huge deal here though I suppose.

Reply:
It would have been a bigger concern for the old `COModifiers`, because they had statefulness in that they modified the models' state while active.
With this? If it's not on the list, it's not getting modified.

Reply:
It's arguably better practice not to have an internal object as part of the public API, but we do that elsewhere anyway.

I guess part of my objection is just that after creating a filter I'd naturally look for an "add..." method and then have to figure out to use `instances` directly instead. Discoverability is lower with this approach.

Not a hill I'm gonna die on though. If I care that much, I might just add the function later.